src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -474,7 +475,7 @@ private Step CreateStep(Step jsonStepInfo, HostControlInfo hostControlInfo, int
                         Sequence = jsonStepInfo.Sequence,
                         RatingTextTitle = formattedText.ToString(),
                         StepType = Step.StepTypes.SURVEY,
-                        IsRatingVisible = dynamoViewModel.Model.PreferenceSettings.IsADPAnalyticsReportingApproved,
+                        IsRatingVisible = !Analytics.DisableAnalytics && Analytics.IsEnabled,
```

Comment:
will `Analytics.IsEnabled` always be true?

Reply:
IsEnabled is false when the DynamoAnalyticsClient is not created. DynamoAnalyticsClient is created when dynamo model is started with these flags
```
!IsServiceMode && !areAnalyticsDisabledFromConfig && !Dynamo.Logging.Analytics.DisableAnalytics
```

Reply:
There might be some other conditions depending on how the Dynamo model is created